<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonNext">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonRun">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="buttonNextInto">
     <property name="text">
      <string>Next Into</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="buttonFinish">
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QTableView" name="contextFrameView"/>
   </item>
   <item row="2" column="2" colspan="4">
    <widget class="QListView" name="stackTraceView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
